tools/commit_slider/case_1.md

```diff
@@ -0,0 +1,316 @@
+# Case #1. Cross-checking of performance degradation.
+#### Themes: Mode and Template customization, API design
+  
+
+For 2 given versions, defined by commit hashes and 2 models we need to resolve which of two changes (ov or model) caused degradation. I.e. we need to differentiate 2 possible cases:
+model-caused
+|'Bad' model | model_1.xml| model_2.xml|
+| --------     | -------    | -------|
+| ov_version_1 | 100 % (ref)|50%     |
+| ov_version_2 | 98 %       |51%     |
+and ov - caused
+|'Bad' ov-version| model_1.xml| model_2.xml|
+| --------     | -------    | -------|
+| ov_version_1 | 100 % (ref)|98%     |
+| ov_version_2 | 48% %       |51%     |
+
+## Version 1. Reusing of existing classes
+
+### Traversal
+We need full bypass of commit set (pair of commits in the given case), ignoring the result of a specific commit.
```

Comment:
"ignoring the result of a specific commit" is not clear. Could you please reformulate or elaborate?